hydra-cluster/src/CardanoClient.hs

```diff
@@ -79,23 +79,23 @@ submitTx :: RunningNode -> Tx -> IO ()
 submitTx RunningNode{networkId, nodeSocket} =
   submitTransaction networkId nodeSocket
 
-waitForPayment ::
+waitForPayments ::
   NetworkId ->
   SocketPath ->
   Lovelace ->
   Address ShelleyAddr ->
   IO UTxO
-waitForPayment networkId socket amount addr =
+waitForPayments networkId socket amount addr =
```

Comment:
Should: document the reason why you renamed this

You seemingly changed the name to indicate this may also return multiple UTxO? You should document that for the future reader.